flight/Modules/Airspeed/airspeed.c

```diff
@@ -104,10 +104,10 @@ static int8_t airspeedADCPin=-1;
 // Private functions
 static void airspeedTask(void *parameters);
 void baro_airspeedGet(BaroAirspeedData *baroAirspeedData, uint32_t *lastSysTime, uint8_t airspeedSensorType, int8_t airspeedADCPin);
-static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev);
+static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev, void *ctx, void *obj, int len);
```

Comment:
@mlyle can you elaborate on this? 

1. Isn't a good argument IMO; yes it results in more changes but also cleaner code in the long term.
2. Why is it really important?
3. It's true that up to 4 arguments can be passed using registers, but this doesn't mean one should always use  4. It just means the ARM designers decided people rarely use more than 4 arguments, but using fewer than 4 arguments is still better.